packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -114,6 +114,31 @@ type ActionListSectionProps = {
   _sectionChildValues?: string[];
 } & TestID &
   DataAnalyticsAttribute;
+
+const _ActionListSectionTitle = ({
+  title,
+  isInsideVirtualizedList = false,
+}: {
+  title: string;
+  isInsideVirtualizedList?: boolean;
+}): React.ReactElement => {
+  return (
+    <StyledActionListSectionTitle
+      {...makeAccessible({
+        hidden: !isInsideVirtualizedList,
```

Comment:
why hide?

Reply:
in the case of a normal heading, we're announcing the title as the group label, so we have maked this as hidden. 
 this code was already their have moved it bit ahead and have done VirtualizedList changes.